lib/karma/KarmaServerManager.js

```diff
@@ -167,4 +142,50 @@ KarmaServer.prototype.isKilled = function() {
     return this._status === KarmaServerStatus.KILLED;
```

Comment:
This might need some changes as well if we decide to introduce the new STOPPED status.